aws-xray/src/test/java/io/opentelemetry/contrib/awsxray/SamplingRuleApplierTest.java

```diff
@@ -72,6 +74,15 @@ class ExactMatch {
             .put(AttributeKey.longKey("speed"), 10)
             .build();
 
+    private final Attributes newSemCovAttributes =
```

Comment:
we usually call it stable semconv

Reply:
changed abrv to `semconv` instead of `semcov`